gammapy/background/energy_offset_array.py

```diff
@@ -0,0 +1,135 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import numpy as np
+from astropy.units import Quantity, UnitsError
+from astropy.coordinates import Angle
+from astropy.table import Table
+from astropy.units import Quantity
+from astropy.coordinates import Angle
+
+__all__ = [
+    'EnergyOffsetArray',
+]
+
+
+class EnergyOffsetArray(object):
+    """
+    Energy offset dependent array.
+
+    Parameters
+    ----------
+    energy : `~astropy.units.Quantity`
+         energy bin vector
+    offset : `~astropy.coordinates.Angle`
+        offset bin vector
+    data : `~numpy.ndarray`
+        data array (2D)
+
+    """
+
+    def __init__(self, energy=None, offset=None, data=None):
+        self.energy = Quantity(energy, 'TeV')
+        self.offset = Angle(offset, 'deg')
+        self.data = data
+
+    def fill_events(self, event_list):
+        """
+        from the fill_events method of the Cubebackgroundmodel class in model.py
```

Comment:
This docstring needs to be updated. It can be very simple and short, e.g.:
```
"""Fill events histogram.

This add the counts to the existing value array.
"""